test/integration/user-management-api/service_subscriptions_controller_test.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Admin::Api::ServiceSubscriptionsControllerTest < ActionDispatch::IntegrationTest
+
+  SERVICE_SUBSCRIPTION_ATTRIBUTES = %w[id plan_id service_id user_account_id created_at updated_at state paid_until trial_period_expires_at setup_fee type variable_cost_paid_until tenant_id].freeze
+
+  def setup
+    @service_plan     = FactoryBot.create(:service_plan, issuer: service)
+    @application_plan = FactoryBot.create(:application_plan, issuer: service)
+
+    @buyer            = FactoryBot.create(:buyer_account, provider_account: current_account)
+    @service_contract = FactoryBot.create(:simple_service_contract, plan: @service_plan, user_account: @buyer)
+
+    @buyer.buy! @application_plan
+
+    @token = FactoryBot.create(:access_token, owner: current_account.admin_users.first!, scopes: 'account_management').value
+    host! current_account.internal_admin_domain
+  end
+
+  attr_reader :buyer, :token, :service_contract
+
+  class ProviderAccountTest < Admin::Api::ServiceSubscriptionsControllerTest
+    test 'index' do
+      get admin_api_account_service_subscriptions_path(account_id: buyer.id, format: :json, access_token: token)
+      assert_response :ok
+
+      json_body = JSON.parse(response.body)
+      assert_equal service_contract.id, json_body.dig('service_subscriptions', 0, 'service_subscription', 'id')
```

Comment:
Maybe also assert the number of items in the array is correct?

Reply:
How about this? https://github.com/3scale/porta/pull/3759/commits/e24dede96c2251a9fc3c0e56452e70b97c9ec38d#diff-9259541b1220a7726288726b4c22fb87aa231de82280094ee00addd29f114b40R33

I didn't like verifying a single element.

Reply:
Nice